src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherAccountingLineTotalsValidation.java

```diff
@@ -64,6 +68,21 @@ public boolean validate(AttributedDocumentEvent event) {
                 return true; 
             }
         }
+        
+        //check accounting line totals equal DV total amount
```

Comment:
maybe instead of this comment, this block could be extracted into a method with a meaningful name